rs/rust_canisters/backtrace_canister/BUILD.bazel

```diff
@@ -0,0 +1,55 @@
+load("@rules_rust//rust:defs.bzl", "rust_test", "rust_binary")
+load("//bazel:canisters.bzl", "rust_canister")
+
+package(default_visibility = ["//visibility:public"])
+
+DEPENDENCIES = [
+    # Keep sorted.
+    "@crate_index//:candid",
+    "@crate_index//:ic-cdk",
+]
+
+MACRO_DEPENDENCIES = [
+    # Keep sorted.
+    "@crate_index//:ic-cdk-macros",
+]
+
+DEV_DEPENDENCIES = []
+
+MACRO_DEV_DEPENDENCIES = []
+
+ALIASES = {}
+
+rust_canister(
+    name = "backtrace-canister",
+    srcs = ["src/main.rs"],
+    aliases = ALIASES,
+    proc_macro_deps = MACRO_DEPENDENCIES,
+    service_file = ":backtrace_canister.did",
+    deps = DEPENDENCIES,
+)
+
+rust_test(
+    name = "backtrace_canister_test",
+    srcs = ["src/main.rs"],
+    aliases = ALIASES,
+    data = ["backtrace_canister.did"],
+    env = {
+        "DID_PATH": "rs/rust_canisters/backtrace_canister/backtrace_canister.did",
+    },
+    proc_macro_deps = MACRO_DEPENDENCIES + MACRO_DEV_DEPENDENCIES,
+    deps = DEPENDENCIES + DEV_DEPENDENCIES,
+)
+
+rust_binary(
```

Comment:
Where do we need this binary? Isn't the canister binary enough?

Reply:
The native binary can be used to produce the candid interface: https://github.com/dfinity/ic/pull/1353/files#diff-4aafc3e8a7ad2a6fce382ce284b5510c40685f0103a20b88be8d6d88aff963d1R59-R62